docs/AboutUs.md

```diff
@@ -13,29 +13,30 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 <img src="images/johndoe.png" width="200px">
 
-[homepage](http://www.comp.nus.edu.sg/~damithch)
-[github](https://github.com/johndoe)
-[portfolio](team/johndoe.md)
+[[homepage](http://www.comp.nus.edu.sg/~damithch)]
+[[github](https://github.com/johndoe)]
+[[portfolio](team/johndoe.md)]
 
 * Role: Project Advisor
 
 ### Mohd Shafik
 
 <img src="images/ad-nap.png" width="200px">
 
-[github](https://github.com/AD-NAP/)
-[portfolio](team/ad-nap.md)
+[[github](https://github.com/AD-NAP/)]
```

Comment:
```suggestion
[github](https://github.com/AD-NAP/)
```

Reply:
If i not wrong thats just so the hyper link is wrapped by a set of [] when viewing the markdown. 
I tested it does not display properly on IntelliJ but does work on Github

Reply:
If I am not wrong that is just so the hyper link is wrapped by a set of [] when viewing the markdown on GitHub for readability. 
I tested it does not display properly on IntelliJ but does work on GitHub.

Reply:
Right, I just noticed it now. I think we can leave the [] enclosing like the original file. We'll clarify with TA next time 😄